code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2084,4 +2084,171 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	taste_description = "a sweet and sour slice of apple"
 	glass_icon_state = "jack_rose"
 	glass_name = "Jack Rose"
-	glass_desc = "Enough of these, and you really will start to suppose your toeses are roses."
\ No newline at end of file
+	glass_desc = "Enough of these, and you really will start to suppose your toeses are roses."
+
+/datum/reagent/consumable/ethanol/turbo
+	name = "Turbo"
+	id = "turbo"
+	description = "A turbulent cocktail associated with outlaw hoverbike racing. Not for the faint of heart."
+	color = "#e94c3a"
+	boozepwr = 85
+	quality = DRINK_VERYGOOD
+	taste_description = "the outlaw spirit"
+	glass_icon_state = "turbo"
+	glass_name = "Turbo"
+	glass_desc = "A turbulent cocktail for outlaw hoverbikers."
+
+/datum/reagent/consumable/ethanol/turbo/on_mob_add(mob/living/M)
+	..()
+	M.add_movespeed_modifier(id, update=TRUE, priority=100, multiplicative_slowdown=-0.20, blacklisted_movetypes=(FLYING|FLOATING))
```

Comment:
lol

Reply:
I need a good reason to keep this in, speed boosters are mega cringe.

Reply:
> 
> 
> I need a good reason to keep this in, speed boosters are mega cringe.

The speed boost is the mildest in the game, ionly 1/4th of the much easier to make and stronger nuka cola. The problem with speed boosters is that they are all too fast, we have not really seen this type of speed booster before. Its high alcohol content limits extended usage.

Reply:
I thought I btfo'd nuka cola huh

Reply:
I have an idea of something that might be more unique for this drink instead of the speedboost.

Reply:
I have removed the speed boost and added a more interesting mechanic.